lib/internal/test_runner/mock/fake_timers.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
```

Comment:
I think we should emit an experimental warning when this file is loaded, at least for a few releases.